README

```diff
@@ -15,74 +25,74 @@ Evaluation metric currently in xapian
  * Precision at Recall
  * Recall
 
-Indexing Collection with evaluation Module.
+Indexing Collection with evaluation Module
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 User can index a collection using
 ::
- . ./trec_index configuration_file
+ . ./trec_index <configuration_file>
```

Comment:
This won't work as currently documented since `.` sources a shell script (see `help .` in bash) but `./trec_index` is a binary file:

```
olly@gemse:~/git/xapian-evaluation/bin$ . ./trec_index 
bash: .: ./trec_index: cannot execute binary file
```

Also, I don't think it's helpful to add `<`...`>` around placeholders like this, since this renders as a literal block and so suggests these characters should be literally included in the command.

I think it would be clearer to just make these examples of the actual commands to run - i.e. use what one would typically call the configuration file in the example command.

Reply:
We ship an example configuration file called `config`, so that would be a good filename to use in these examples of how to run the programs.

Reply:
I have corrected the command to `trec_index config`, etc.